In [1]:
import scipy.stats

# Lab | Hypothesis Testing

Instructions

It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?



1) Set up the hypothesis test.

Ho: μ =120 mm Hg

Ha: μ != 120 mm Hg

2) Write down all the steps followed for setting up the test.

It is a two-tail test

Fix the confidence level. = 95 %

Significance level a=0.025

3) Calculate the test statistic by hand and also code it in Python. It should be 4.76190. What decision can you make based on this calculated value?

standard deviation for the sample stddev = 21.21 mm Hg
mean systolic blood presure of the sample = 130.1 mm Hg
mean systolic blood pressure of population μ = 120 mm Hg
sample size of n = 100 people

t = (x-μ)/ stddev/10

t = (130.1-120)/ 21.21/10

t = 4.7619


In [4]:
#tc = +-scipy.stats.t.ppf(1-(0.05/2)=0.975, df=99)
tc = +-scipy.stats.t.ppf(0.975, df=99)
tc

-1.9842169515086827

The test statistic is not in the allowed region

Our t = 4.7619 > + -1.9842 =tc-> I rejected the null hypothesis